### DISCUSSION TITLE
How to set nvim-lspconfig -> tsserver to prefer alias imports

### DISCUSSION THREAD

**Discussion by rykugur** [LazyVim/LazyVim#2564] [Q&A]
I'm relatively new to neovim/lazyvim, and must admit it isn't immediately clear how to override this.

I have the following alias setup in `tsconfig.json`:

```
...
    "paths": {
      "@types": [
        "./src/types/index.ts"
      ],
    },
...
```

And this works fine if I manually import. However, if I try to use autocomplete, it uses relative path import. Here's my plugin lua file where I'm attempting to override the tsserver settings:

```
return {
	"neovim/nvim-lspconfig",
	opts = {
		servers = {
			tsserver = {
				init_options = {
					preferences = {
						importModuleSpecifierPreference = "non-relative",
					},
				},
			},
		},
	},
}
```

Which I pulled off [these docs](https://github.com/typescript-language-server/typescript-language-server/blob/master/docs/configuration.md#preferences-options). Can somebody tell me what I've done wrong and perhaps help me wrap my head around this?

**Comment by ArthurTimofey** [comment]
Bumping this since i too am running into this issue, it only shows absolute imports

Edit: I am using bun.js so a fix for me was to declare `baseUrl` in `tsconfig.json`, i set it to `"./"`
